index.d.ts

```diff
@@ -4,6 +4,17 @@ export default IAxiosRetry;
 export as namespace axiosRetry;
 declare const IAxiosRetry: IAxiosRetry.AxiosRetry;
 
+export type IAxiosRetryConfig = IAxiosRetry.IAxiosRetryConfig;
+export type IAxiosRetryConfigExtended = IAxiosRetry.IAxiosRetryConfigExtended;
+export type IAxiosRetryReturn = IAxiosRetry.IAxiosRetryReturn;
+
+export function isNetworkError(error: Error): boolean;
+export function isRetryableError(error: Error): boolean;
+export function isSafeRequestError(error: Error): boolean;
+export function isIdempotentRequestError(error: Error): boolean;
+export function isNetworkOrIdempotentRequestError(error: Error): boolean;
+export function exponentialDelay(retryNumber?: number, error?: Error, delayFactor?: number): number;
```

Comment:
Those were directly exported before. Those were exported through `AxiosRetry` which should be exported instead.

I'd still want you to answer https://github.com/softonic/axios-retry/pull/250#discussion_r1393768455 more specifically because I'm not sure why the `export =` syntax doesn't work for you.

Reply:
> Those were not exported directly before. Those were exported through AxiosRetry which should be exported instead.

This is to conform to the specification in JavaScript. It is also possible to name import the `isNetworkError` function as follows.
This time, it has been modified accordingly.

![image](https://github.com/softonic/axios-retry/assets/79501292/baf39b52-da36-4c71-8519-23551cf035b5)

Reply:
Well, you can export those since it's a "new feature" but you also have to export `AxiosRetry` since it was exported before and it was possible to do `import { AxiosRetry } from 'axios-retry'` while it's not possible with those changes.